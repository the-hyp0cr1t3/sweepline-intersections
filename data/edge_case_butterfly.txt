4
1 5 9 1
3 5 4 3
5 3.5 3 2
2 2 8 2