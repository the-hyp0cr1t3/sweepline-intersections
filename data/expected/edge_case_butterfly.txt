4
3.0000000000 2.0000000000 3 4
3.6666666667 3.6666666667 1 2
4.6000000000 3.2000000000 1 3
7.0000000000 2.0000000000 1 4

